### DISCUSSION TITLE
revealjs: how to set title and hide the first title slide

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#7152] [Q&A] [answered]
### Description

Hi,
in my qmd file I have a title:

~~~~
---
format:
  revealjs:
    title: "My title"
    theme: [dark, "risorse/custom.scss"]
---
~~~~

For me it's useful because it gives the HTML title `<title>My title</title>`, but I do not want the slide output.
Is there any way to set a title, without having the title slide output?

Thank you

![image](https://github.com/quarto-dev/quarto-cli/assets/30607/5081d951-d9f8-4fcc-b3d1-c4794d2adcf2)

**Comment by royfrancis** [comment, marked as answer]
How about using `pagetitle`?

```
---
format:
  revealjs:
    pagetitle: "My title"
---
```

**Reply by aborruso** [reply]
Thank you very much
